diff --git a/config.def.h b/config.def.h
index 0e01717..b9e4647 100644
--- a/config.def.h
+++ b/config.def.h
@@ -42,8 +42,8 @@ static unsigned int tripleclicktimeout = 600;
 int allowaltscreen = 1;
 
 /* frames per second st should at maximum draw to the screen */
-static unsigned int xfps = 120;
-static unsigned int actionfps = 30;
+static unsigned int xfps = 1001;
+static unsigned int actionfps = 1001;
 
 /*
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
